src/client/reducers/index.js

```diff
@@ -31,7 +30,10 @@ const initialState = {
   score: 0,
   gamesList: [],
   game: null,
-  nickname: 'defaultName',
+  player: {
+    id: null,
+    nickname: 'Player'
+  },
   isNicknameModalOpen: false,
   players: [{ nickname: 'Me', webRTCId: 0, score: 0, spectrum: initSpectrum() }, { nickname: 'You', webRTCId: 1, score: 0, spectrum: initSpectrum() }],
```

Comment:
should be removed

Reply:
heu non, je viens de l'ajouter justement !

Reply:
nan le players: [{ nickname: 'Me', webRTCId: 0, score: 0, spectrum: initSpectrum() }, { nickname: 'You', webRTCId: 1, score: 0, spectrum: initSpectrum() }].
ca c'est obsolete mnt c'est ds game.players
mais l'interface github porte a confusion ouais

Reply:
ha oui. too late j'ai deja merge, je clean ca a la prochaine PR